src/reactionsystem.jl

```diff
@@ -1371,7 +1382,7 @@ Notes:
   differential equations.
 """
 function Base.convert(::Type{<:SDESystem}, rs::ReactionSystem;
-                      noise_scaling = nothing, name = nameof(rs),
+                      noise_scaling = get_noise_scaling(rs), name = nameof(rs),
```

Comment:
```suggestion
                      noise_scaling = nothing,
```
given that you later check if it is `nothing` and set it to this anyways?

Reply:
This basically has to do with the fact that we are proofing for a deprecated feature. In principle, the kwarg should be `noise_scaling = get_noise_scaling(rs)`, and `SDEProblem` should not submit any `noise_scaling` value. However, since we previously allowed `noise_scaling` in `SDEProblem`, we have to keep this functionality running. 

I let it be `noise_scaling = get_noise_scaling(rs)` since that is the future version, thinking it would make things easier when we deprecate things. However, I will change it to nothing and add a comment about it to the code.